docs/pages/choose-an-edition/teleport-team.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: "Introduction to Teleport Team"
+description: "Teleport Team makes it quick and easy for small organizations set up secure RBAC for their infrastructure."
+tocDepth: 3
+---
+
+Teleport Team helps small organizations get started quickly with enabling secure
+access to their infrastructure. 
+
+With Teleport Team, the Auth Service and Proxy Service are fully managed, giving
+you a scalable, fault tolerant certificate authority that you can use to
+configure access from users across the globe to your infrastructure, including
+private networks.
+
+After you start a [free trial](https://goteleport.com/signup) of Teleport Team,
+you can set up RBAC, enable single sign-on with GitHub or Teleport's own
+identity provider, and start protecting resources with Teleport.
+
+```mermaid
+flowchart LR
+subgraph team["Teleport Team tenant"]
+  direction LR
+  proxy["Teleport Proxy Service"]
+  auth["Teleport Auth Service"]
+  proxy<-.->auth
+end
+
+subgraph public["The internet"]
```

Comment:
Consider giving this an empty title and no background. "The internet" might be best diagramed as open.